docs/tutorials/detecting_data_drift/univariate_drift_detection.rst

```diff
@@ -9,7 +9,7 @@ Just The Code
 -------------
 
 .. nbimport::
-    :path: ./example_notebooks/Tutorial - Drift - Univariate.ipynb
+    :path: ./example_notebooks/Tutorial - Drift - Univariate - Car Loan.ipynb
```

Comment:
I just noticed we still have `integer` values in the `y_pred` column. That forces us to explicitly set that column as categorical before running NannyML. This was one of the things we didn't want to have any longer in the tutorials. I must've missed that one, sorry.

This does impact some of the notebooks, so I'll make the change to the dataset and the relevant notebooks tomorrow (remove the explicit conversion to `category` datatype)